.storybook/addons.js

```diff
@@ -1,2 +1,3 @@
+import '@storybook/addon-actions';
 import '@storybook/addon-actions/register';
```

Comment:
It seems unnecessary to import both @storybook/addon-actions and @storybook/addon-actions/register, but I might be wrong. Could you explain this?

Reply:
Actions addon places a console area at the bottom of the page so the developer can see what objects/strings are being passed when you click on a button etc. As for the `/register` import, that was already there, so I didn't want to touch it

Reply:
That makes sense, thanks!